README.md

```diff
@@ -69,10 +70,12 @@ pip3 install gattlib
 If for some reason the gattlib installation fails:
 
 ```shell
+apt-get install pkg-config python3-dev
 pip3 download gattlib
 tar xvzf ./gattlib-0.20150805.tar.gz
 cd gattlib-0.20150805/
-sed -ie 's/boost_python-py34/boost_python-py35/' setup.py
+sed -ie 's/boost_python-py34/boost_python-py36/' setup.py # "py36" might be "py37" (for example). Check "python3 --version"
```

Comment:
It's all seems a bit crazy. I was that close to giving up before I came across an obscure post mentioning boost_python3.

Reply:
> This can probably be made more generic ...
Makes sense- good idea.  It would need to be "version=$( python3 -c ...." on my system, because "python" gets you python 2.